lib/python/rose/upgrade.py

```diff
@@ -203,8 +203,8 @@ def change_setting_value(self, config, keys, value, forced=False,
         node = config.get([section, option])
         if node is None:
             if forced:
-                return self.add_setting(config, keys, value, node.state,
-                                        comments, info)
+                return self.add_setting(config, keys, value=value,
```

Comment:
Missing because it has no state, so is safely defaulting to `None` which means 'enabled'.